code/src/vs/workbench/api/common/extHostExtensionActivator.ts

```diff
@@ -170,6 +170,9 @@ export class ExtensionsActivator implements IDisposable {
 	private readonly _externalExtensionsMap: Map<string, ExtensionIdentifier>;
 	private readonly _host: IExtensionsActivatorHost;
 	private readonly _operations: Map<string, ActivationOperation>;
+	private readonly _excludedDependencies = [
+		'vscode.github-authentication',
```

Comment:
Does it make the previously added exclusion mechanism redundant?
https://github.com/che-incubator/che-code/commit/7aa9f31489c0dcbe38841e4204770aa6b6128bf2#diff-36fae229a2323a58cfac9a248b606846fd67fbf795dd4f8aecacfd6007faf134R416